acceptance/Rakefile

```diff
@@ -83,11 +83,12 @@ namespace :ci do
 
       t.add_env(:name => 'SHA',           :message => 'The sha for pe-client-tools')
       t.add_env(:name => 'SUITE_VERSION', :message => 'The suite version used by Jenkins')
+      t.add_env(:name => 'PE_FAMILY',     :message => 'The puppet enterprise major branch to install from')
 
       t.add_env do |env|
         env.name = 'pe_dist_dir'
         env.message = 'The location to download PE from example "http://neptune.puppetlabs.lan/20XX.X/ci-ready"'
-        env.default = 'http://neptune.puppetlabs.lan/2016.3/ci-ready'
+        ENV['pe_dist_dir'] ||= "http://neptune.puppetlabs.lan/#{ENV['PE_FAMILY']}/ci-ready"
```

Comment:
Can we take up the conversation of removing the environment variable separately? This should be coordinated across the client tools components and raises the question about the value of being able to over-ride the location for the `pe_dist_dir`. This variable can have value when testing frankenbuilds.